### ISSUE TITLE
Unable to process command '::add-path::...'

### ISSUE THREAD

**Issue by Qard** [microsoft/setup-msbuild#84]
This action seems to be using unsecure commands which are rejected by default. Is there some way around that which doesn't involve disabling security?

```
Error: Unable to process command '::add-path::C:\Program Files\Microsoft Visual Studio\2022\Enterprise\MSBuild\Current\Bin' successfully.
Error: The `add-path` command is disabled. Please upgrade to using Environment Files or opt into unsecure command execution by setting the `ACTIONS_ALLOW_UNSECURE_COMMANDS` environment variable to `true`. For more information see: https://github.blog/changelog/2020-10-01-github-actions-deprecating-set-env-and-add-path-commands/
```

**Comment by timheuer** [comment]
@Qard this was fixed a while back (see #26) -- are you using the latest version?

**Comment by Qard** [comment]
Oh, whoops. Disregard. 😅
